### PR TITLE
Initial commit for results. WIP

### PR DESCRIPTION
**Pull request by catileptic**
### Requirements for making a pull request

Thank you for contributing to our project! 

Please fill out the template below to help the project maintainers review it as fast as possible and include your contribution to the project.

### What does it fix?

Closes 

Please mention the main changes this PR brings.

### How has it been tested?

Please describe the tests that you ran to verify your changes.

### CODE REVIEW

File: civil_society_vote/hub/templates/candidate/results.html [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+                        {% endif %}
+                    </div>
+                    <div class="need-title is-hidden-tablet">{{candidate.name}}</div>
+                </div>
+                <div class="need-subtitle is-hidden-mobile">{{candidate.org}}</div>
+                <div class="need-title is-hidden-mobile">
+                    <a class="has-text-black" href="{% url 'candidate-detail' candidate.pk %}">{{candidate.name}}</a>
+                </div>
+                <!-- <div class="need-description is-hidden-mobile">{{candidate.experience}}</div> -->
+                <div class="need-title">Voturi: 0</div>
```
Comment (catileptic) [inline civil_society_vote/hub/templates/candidate/results.html:71]: Query goes here

**Comment by catileptic** [comment]
image

File: civil_society_vote/civil_society_vote/settings/prod.py [hunk 2]
```diff
@@ -13,3 +13,5 @@
 
 STATIC_ROOT = os.path.join(BASE_DIR, "../", "static")
 STATICFILES_DIRS = []
+
+CACHES = {"default": {"BACKEND": "django.core.cache.backends.locmem.LocMemCache", "LOCATION": "unique-snowflake",}}
```
Comment (aramboi) [inline civil_society_vote/civil_society_vote/settings/prod.py:17]: You can name the `LOCATION` however you like. This is fine though. :)

File: civil_society_vote/hub/views.py [hunk 3]
```diff
@@ -355,6 +355,38 @@ def get_context_data(self, **kwargs):
...
+    allow_filters = ["domain"]
+    paginate_by = 9
+    template_name = "candidate/results.html"
+
+    def get_qs(self):
+        return Candidate.objects_with_org.filter(
+            org__status=Organization.STATUS.accepted, status=Candidate.STATUS.accepted, is_proposed=True
+        )
+
+        return Candidate.objects_with_org.none()
```
Comment (aramboi) [inline civil_society_vote/hub/views.py:368]: This will never be called.

File: civil_society_vote/hub/templates/candidate/results.html [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+{% endif %}
+</div>
+
+{% endblock %}
+
+{% block content %}
+{% with current_url=request.get_full_path %}
+{{ block.super }}
+{% endwith %}
+{% endblock %}
```
Comment (aramboi) [inline civil_society_vote/hub/templates/candidate/results.html:111]: Missing new line at eof.

**Pull request merged**